lib/CustomFields/pages/EditCustomFieldsSettings/EditCustomFieldsSettings.js

```diff
@@ -160,6 +180,18 @@ const EditCustomFieldsSettings = ({
 
   const formatCustomFieldsForBackend = customFieldsFormData => {
     return customFieldsFormData.customFields.map(({ values: { hidden, ...customFieldData } }) => {
+      const fieldHasOptions =
+        customFieldData.type === fieldTypes.RADIO_BUTTON_SET ||
+        customFieldData.type === fieldTypes.SELECT;
+
+      if (fieldHasOptions) {
+        if (customFieldData.selectField.defaults === NO_DEFAULT_OPTIONS_VALUE) {
+          customFieldData.selectField.defaults = [];
+        } else {
+          customFieldData.selectField.defaults = [customFieldData.selectField.defaults];
```

Comment:
Is `customFieldData.selectField.defaults` an array? Do we need to do destructuring here?

Reply:
Since we use radion buttons to chose the default option, final-form stores the data as a plain value: `{ defaults: 'option' }`. Backend expects us to send an array of default options:  `{ defaults: ['option'] }`, that's why I'm doing these transformations